Free-Code-Camp-JavaScript-Style-Guide.md

```diff
@@ -67,12 +68,12 @@ if(true)
 }
 ```
 
-## Else
+## else

```

Comment:
just because more than half of the headings were lowercase on other sections so I made them consistent.  And, the "else" keyword is lowercase in javascript.